orchagent/orchdaemon.cpp

```diff
@@ -37,6 +37,8 @@ bool OrchDaemon::init()
 {
     SWSS_LOG_ENTER();
 
+    string platform = getenv("platform") ? : "";
```

Comment:
Omitting the middle operand is only allowed via a GCC extension; it's not part of the standard. We don't currently intend to change compilers, but this isn't very portable, and since it's only called once in OrchDaemon::init(), it's only saving a few cycles at startup. Consider changing to string platform = getenv("platform") ? getenv("platform") : "";

Reply:
Done